indico/modules/rb/client/js/modules/admin/MapAreasPage.jsx

```diff
@@ -0,0 +1,366 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {Form as FinalForm} from 'react-final-form';
+import Leaflet from 'leaflet';
+import {FeatureGroup, Map, Rectangle, TileLayer, Tooltip, ZoomControl} from 'react-leaflet';
+import {EditControl} from 'react-leaflet-draw';
+import {Button, Form, Header, Modal} from 'semantic-ui-react';
+
+import {FinalCheckbox, FinalInput, FinalSubmitButton} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import {selectors as mapSelectors, actions as mapActions} from '../../common/map';
+import {selectors as configSelectors} from '../../common/config';
+
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-draw/dist/leaflet.draw.css';
+import './MapAreasPage.module.scss';
+
+
+Leaflet.EditToolbar.Delete.include({
+    removeAllLayers: false
+});
+
+const DEFAULT_CENTER = [46.233832398, 6.053166454];
```

Comment:
Is it OK to keep this hardcoded?

Reply:
Probably not amazing. Could we query the tile server for the center of the area it has available somehow? @pferreir

Reply:
Yep, not the best thing to do. But couldn't find a way to get that information from the tile server